### PR TITLE
EVA-2056 - Use updated OpenCGA library that uses Mongo 3.8 driver

### PR DESCRIPTION
**Pull request by sundarvenkata-EBI**
EvaPipelineBatchConfigurer.java - More ceremony needed because of [change in the default serialization mechanism to Jackson](https://stackoverflow.com/a/49636389/2601814).

### CODE REVIEW

**Comment by sundarvenkata-EBI** [comment]
CI ran [successfully](https://www.travis-ci.com/github/sundarvenkata-EBI/eva-pipeline/builds/214253284). For some reason, the build status does not seem to show up in Github.

**Comment by tcezard** [comment]
Does this PR needs extra review since it was accepted and that the test passes now ?

**Comment by sundarvenkata-EBI** [comment]
> Does this PR needs extra review since it was accepted and that the test passes now ?

Yes, I have outlined the changes made in the commits subsequent to the previous approval.

File: src/test/java/uk/ac/ebi/eva/test/utils/JobTestUtils.java [hunk 1]
```diff
@@ -42,10 +42,7 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.InputStreamReader;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Set;
-import java.util.TreeSet;
+import java.util.*;
```
Comment (andresfsilva) [inline src/test/java/uk/ac/ebi/eva/test/utils/JobTestUtils.java:45]: please make the imports explicit

**Review by andresfsilva — APPROVED** [review]

**Pull request merged**